projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -46,6 +46,7 @@ export class SearchBoxComponent {
    * for example when we click inside the search result section.
    */
   private ignoreCloseEvent = false;
+  public chosenWord = '';
```

Comment:
This should be private, per our coding guidelines - https://sap.github.io/spartacus-docs/coding-guidelines/#services

Reply:
since this member needs to be accessible in the template, it cannot be private.